apps/challenges/views.py

```diff
@@ -1255,16 +1258,30 @@ def get_or_update_challenge_phase_split(request, challenge_phase_split_pk):
     """
     challenge_phase_split = get_challenge_phase_split_model(
         challenge_phase_split_pk)
+    challenge_phase_id = ChallengePhaseSplit.objects.filter(pk=challenge_phase_split_pk).values_list(
+        'challenge_phase_id', flat=True)[0]
+    challenge_id = ChallengePhase.objects.filter(pk=challenge_phase_id).values_list('challenge_id', flat=True)[0]
+
```

Comment:
@nileshprasad137 There is no need to query the database again. You can simply get the challenge id from challenge phase using `challenge_id = challenge_phase_split.challenge_phase.challenge.id`

Reply:
Oh yes! I missed it. I'll fix them.